tests/concat.rs

```diff
@@ -0,0 +1,148 @@
+use predicates::prelude::*;
+use snapbox::cmd::{cargo_bin, Command};
+use std::path::Path;
+
+pub mod common;
+use crate::common::*;
+
+#[path = "../src/io.rs"]
+pub mod io;
```

Comment:
Might be able to put this after `use super::*` below and avoid the path line, but I'm not 100% sure